tests/fixtures/deprecation_warning_update_attr/extensions.py

```diff
@@ -0,0 +1,8 @@
+from copier_templates_extensions import ContextHook
+
+
+class ContextUpdater(ContextHook):
+    update = "not sentinel"
```

Comment:
Shouldn't this rather be

```suggestion
    update = False
```

to correctly test the previous behavior?

Reply:
The intent is to emit a deprecation warning if the `update` attribute is set, whatever it's value. I wanted to emphasize that. So, sure, we could set it to False too, since that's what users would be doing. No strong opinion ^^

Reply:
Lets go with the current diff then. This code is deprecated and will be removed in 6 months / 1 year anyway. If someone wants to improve typing (#3), be my guest!